miso-web/src/main/webapp/scripts/list_container.js

```diff
@@ -50,7 +50,7 @@ ListTarget.container = {
         },
         ListUtils.idHyperlinkColumn("Last Run Name", "run", "lastRunId",
             function(container) {
-              return "RUN" + container.lastRunId;
+              return (container.lastRunId ? "RUN" + container.lastRunId : null);
```

Comment:
Can you modify the `idHyperlinkColumn` function to not display the link at all when the fetched ID is missing?